Kudu.Core/Jobs/ContinuousJobRunner.cs

```diff
@@ -297,5 +302,26 @@ protected virtual void Dispose(bool disposing)
                 }
             }
         }
+
+        private int GetAvailablePortForWebJob()
+        {
+            if (JobSettings.IsWebHook)
+            {
+                var listener = new TcpListener(IPAddress.Loopback, 0);
+                try
+                {
+                    listener.Start();
+                    return ((IPEndPoint)listener.LocalEndpoint).Port;
+                }
+                finally
+                {
+                    listener.Stop();
```

Comment:
You start/stop a listener here to determine an available port at a moment in time. Will that port remain available/the same?